README.md

```diff
@@ -831,6 +831,23 @@ mrsk lock acquire -m "Doing maintanence"
 mrsk lock release
 ```
 
+## Gradual restarts
+
+When deploying to large numbers of hosts, you might prefer not to restart your services on every host at the same time.
+
+MRSK's default is to start new containers on all hosts in parallel. But you can control this by configuring `group_limit` and `group_wait`.
```

Comment:
As top-level config, these options are pretty generic. It's hard to tell how they apply. Could they live within a new config section for app boot?

Reply:
Yeah, good idea 👍

Reply:
I've moved these under a new `boot` section.